cloudinit/config/cc_mounts.py

```diff
@@ -223,6 +223,76 @@ def suggested_swapsize(memsize=None, maxsize=None, fsys=None):
     return size
 
 
+def get_fstype(path):
+    """Return the type of the filesystem that path is on.
+    If path is a mount point itself, then the type of the filesystem mounted
+    on path rather than its parent dir."""
+    return util.get_mount_info(path)[1]
+
+
+def create_swapfile(fname, size, method="auto"):
```

Comment:
AFAICT, this is never called with `method`<sup>0</sup>. Should we just drop the `method` parameter and simplify the function implementation?

<sup>0</sup> Well, at the moment, it's never called at all. See my comment below. ;-)

Reply:
I agree with the suggestion and I'll elaborate more on the helper on my next commit.